# goal definition
# name of the behavior
string action_block
string optional_data
---
# result definition
string action_block
bool action_block_continue
string message
---
# feedback
string action_block
string action_block_state